core/iwasm/compilation/aot_compiler.h

```diff
@@ -13,29 +13,6 @@
 extern "C" {
 #endif
 
-typedef enum IntCond {
-  INT_EQZ = 0,
-  INT_EQ,
-  INT_NE,
-  INT_LT_S,
-  INT_LT_U,
-  INT_GT_S,
-  INT_GT_U,
-  INT_LE_S,
-  INT_LE_U,
-  INT_GE_S,
-  INT_GE_U
-} IntCond;
-
-typedef enum FloatCond {
-  FLOAT_EQ = 0,
-  FLOAT_NE,
-  FLOAT_LT,
-  FLOAT_GT,
-  FLOAT_LE,
-  FLOAT_GE
-} FloatCond;
-
```

Comment:
Is there any compile error or warning? Do we need to move them and why just move part of the enum types into aot.h?

Reply:
No error or warning found with this patch since if the `aot_compiler.h` included the 'aot.h' had also included.

Move these flags because they will be used both in wamrc and runtime intrinsic (aot_intrinsics.c), some LLVM relative headers are not available on embedded platform.